global-script.lua

```diff
@@ -2000,170 +2000,178 @@ function resetPiece(object, rotation, depth)
     return object
 end
 ------
+-- posOrigin is the base position for placing the island boards.
+-- The tables below include positions relative to this, in order to
+-- allow scaling the board layouts.
+posOrigin = Vector(5.96, 1.08, 16.59)
+scaleFactors = {
+    [true]={name = "Large", position = 1.09, size = 1.1},
```

Comment:
is the name field used anywhere?

Reply:
Not currently. I had originally imagined having multiple sizes (partly this was based on the old setup UI).